client/state/reader/follows/reducer.js

```diff
@@ -20,21 +22,34 @@ import {
  * @param  {Object} action Action payload
  * @return {Array}        Updated state
  */
-export function items( state = [], action ) {
-	switch ( action.type ) {
-		case READER_FOLLOW:
-			return union( state, [ action.url ] );
-
-		case READER_UNFOLLOW:
-			return without( state, action.url );
-
-		case SERIALIZE:
-		case DESERIALIZE:
-			return [];
-	}
-
-	return state;
-}
+export const items = createReducer( {}, {
+	[ READER_FOLLOW ]: ( state, { url } ) => {
+		const urlKey = prepareComparableUrl( url );
+		return {
```

Comment:
should these (FOLLOW and UNFOLLOW) try to preserve any information that's currently present in the key instead of overwriting it with an object that contains just `is_following`?